Dockerfile

```diff
@@ -23,3 +23,14 @@ COPY --from=builder /usr/bin/k6 /usr/bin/k6
 USER 12345
```

Comment:
Not related to this PR, but seeing you've reproduced it below (which we can remove if we change the entrypoint), this would be more readable and maintainable as `USER k6`.

So if you're doing the entrypoint change to the `grafana/k6` image as I suggested above, then let's make this change as well.

In reality, we should update the `builder` stage to not rely on the `GOPATH` workdir, since that's a legacy requirement before Go modules were introduced, but that would be a larger change we don't need to do now.